tests/e2e/cucumber/mocked-responses.spec.ts

```diff
@@ -8,6 +8,14 @@ Given('an intercepted search response', () => {
   cy.intercept(searchEndpoint).as('interceptedResults');
 });
 
+Given('an intercepted search response from {string}', (origin: string) => {
```

Comment:
I think here we can filter from the origin directly:

```suggestion
Given('an intercepted search response from {string}', (origin: string) => {
  cy.intercept({ url: searchEndpoint, query: { origin: `*${origin}*` } }).as(
    `interceptedResultsFrom:${origin}`
  );
});
```